frappe/desk/doctype/desktop_icon/desktop_icon.py

```diff
@@ -149,8 +150,8 @@ def add_user_icon(_doctype, _report=None, label=None, link=None, type='link', st
 				'type': type,
 				'_doctype': _doctype,
 				'_report': _report,
-				'icon': module_icon.icon,
-				'color': module_icon.color,
```

Comment:
maybe just add icon and color in report (?) since this is for report.

Reply:
Added for reports as well:
<img width="1036" alt="screen shot 2018-02-01 at 2 55 52 pm" src="https://user-images.githubusercontent.com/16913064/35671014-21ba6d4c-0760-11e8-8ef6-ea889b627f03.png">
<img width="178" alt="screen shot 2018-02-01 at 2 56 38 pm" src="https://user-images.githubusercontent.com/16913064/35671019-23c6d616-0760-11e8-801e-971d443008a9.png">